NEWDB/Postgres/Data/postgreSQL_schema.sql

```diff
@@ -0,0 +1,39 @@
+DROP DATABASE IF EXISTS reviews;
+
+  CREATE DATABASE reviews;
+  \c reviews;
```

Comment:
Is this a typo "\c" ? or does it mean using that data base in postgres ?

Reply:
yes that is instead of USE database